docs/conf.py

```diff
@@ -28,7 +28,7 @@
 
 # -- Sphinx-Click configuration
 # https://sphinx-click.readthedocs.io/
-extensions.append('sphinx_click')
+extensions.append("sphinx_click")
```

Comment:
We should add the `./docs` to the `SRC_DIRS` in `Makefile`. That way, this file will automatically be linted and formatted in the future. Actually, when I see your changes to `setup.py` below, I'm thinking that we should add this file as well.

Reply:
There are some mypy errors which I do not know how to resolve when I'm trying to add `./docs` to the `SRC_DIRS` . Is it possible if we add it on a different PR?

Reply:
I have done that but there will be a few mypy errors which I would need your feedback on how to resolve it afterwards when I checked in the codes. The pipeline will surely report that failed test.